### PR TITLE
Add Markdown capability globally

### PR DESCRIPTION
**Pull request by talvasconcelos**
Allow to use Markdown notation on LNbits.

Make the description (SITE_DESCRIPTION) capable of being made in Markdown.

Showdown library added to the vendor packages.

Extensions can then use the global `LNbits.utils.convertMarkdown()` method.

### CODE REVIEW

**Comment by talvasconcelos** [comment]
Closes #1912

**Review by dni — APPROVED** [review]

**Comment by arbadacarbaYK** [comment]
Why cant we have a normal textfield that no-coders expect. 
For a chat in Copilot or NostrMarket we need it anyway, isnt it ?

File: lnbits/core/templates/core/index.html [hunk 1]
```diff
@@ -61,7 +61,10 @@ <h5 class="q-my-md">{{SITE_TAGLINE}}</h5>
             ></q-btn>
           </div>
         </div>
-        <p v-else>{{SITE_DESCRIPTION | safe}}</p>
+        <div v-else>
+          <div v-html="formatDescription"></div>
+        </div>
+        <!-- <p v-else>{{SITE_DESCRIPTION}}</p> -->
```
Comment (motorina0) [inline lnbits/core/templates/core/index.html:67]: Remove dead code

File: lnbits/core/static/js/index.js [hunk 2]
```diff
@@ -5,11 +5,17 @@ new Vue({
...
-        data: {}
+        data: {},
+        description: ''
       },
       walletName: ''
     }
   },
+  computed: {
+    formatDescription() {
+      return LNbits.utils.convertMarkdown(this.description)
```
Comment (motorina0) [inline lnbits/core/static/js/index.js:16]: `description` can also be HTML. Is that covered by the converter?

File: lnbits/core/templates/core/index.html (hunk 1 above)
Comment (dni) [inline lnbits/core/templates/core/index.html:67]:
```suggestion
        <!-- <p v-else>{{SITE_DESCRIPTION}}</p> -->
```
```suggestion
```

**Review by arcbtc — APPROVED** [review]

**Pull request merged**